charts/jenkins/CHANGELOG.md

```diff
@@ -12,11 +12,45 @@ Use the following links to reference issues, PRs, and commits prior to v2.6.0.
 The changelog until v1.5.7 was auto-generated based on git commits.
 Those entries include a reference to the git commit to be able to get more details.
 
+## 5.0.0
+### Breaking changes - 5.0.0
```

Comment:
@lemeurherve I assume this will break the changelog parser?

it might be better to create an upgrade guide file and just link to that instead?